stock_picking_mass_action/wizard/mass_action.py

```diff
@@ -0,0 +1,128 @@
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import fields, api
+from openerp.models import TransientModel
+from openerp.tools.translate import _
+
+
+class StockPickingMassAction(TransientModel):
+    _name = 'stock.picking.mass.action'
+    _description = 'Stock Picking Mass Action'
+
+    @api.model
+    def _default_check_availability(self):
+        return self.env.context.get('check_availability', False)
+
+    @api.model
+    def _default_force_availability(self):
+        return self.env.context.get('force_availability', False)
+
+    @api.model
+    def _default_transfer(self):
+        return self.env.context.get('transfer', False)
+
+    @api.model
+    def _default_create_invoice(self):
+        return self.env.context.get('create_invoice', False)
+
+    confirm = fields.Boolean(
+        string='Mark as Todo', default=True,
+        help="""check this box if you want to mark as Todo the"""
+        """ the selected Pickings.""")
+
+    check_availability = fields.Boolean(
+        string='Check Availability', default=_default_check_availability,
+        help="""check this box if you want to check the availability of"""
+        """ the selected Pickings.""")
+
+    force_availability = fields.Boolean(
+        string='Force Availability', default=_default_force_availability,
+        help="""check this box if you want to force the availability"""
+        """ of the selected Pickings.""")
+
+    transfer = fields.Boolean(
+        string='Transfer', default=_default_transfer,
+        help="""check this box if you want to transfer all the selected"""
+        """ pickings.\n You'll not have the possibility to realize a"""
+        """ partial transfer.\n If you want  to do that, please do it"""
+        """ manually on the picking form.""")
+
+    create_invoice = fields.Boolean(
+        'Create Invoices/Refunds', default=_default_create_invoice,
+        help="""check this box if you want to create Invoices or Refunds for"""
+        """ all the selected Pickings.""")
+
+    @api.multi
+    def mass_action(self):
+        self.ensure_one()
+        picking_obj = self.env['stock.picking']
+        transfert_wizard_obj = self.env['stock.transfer_details']
+        picking_ids = self.env.context.get('active_ids')
+
+        # Get draft pickings and confirm them if asked
+        if self.confirm:
+            domain = [('state', '=', 'draft'),
+                      ('id', 'in', picking_ids)]
+            draft_picking_lst = picking_obj.search(domain, order='min_date')
+            draft_picking_lst.action_confirm()
+
```

Comment:
Only tastes, but if you're using already comments, you already have a separator, so no need for a second one